.github/workflows/daml-all-in-one.yaml

```diff
@@ -0,0 +1,56 @@
+name: daml-all-in-one-publish
```

Comment:
@raynatopedrajeta Please rename the yaml file to match this job name. The `-publish` is missing from the filename.